backtester/eventhandlers/statistics/fundingstatistics.go

```diff
@@ -138,13 +143,13 @@ func CalculateFundingStatistics(funds funding.IFundingManager, currStats map[str
 			decimal.NewFromFloat(interval.IntervalsPerYear()),
 			decimal.NewFromInt(int64(len(usdStats.HoldingValues))),
 		)
-		if err != nil {
+		if err != nil && !errors.Is(err, gctmath.ErrPowerDifferenceTooSmall) {
 			return nil, err
 		}
 		usdStats.CompoundAnnualGrowthRate = cagr
```

Comment:
var cagr decimal.Decimal above should hug with gctmatch call, as its being allocated every for iteration.